### DISCUSSION TITLE
Welcome to vuejs-clipper Discussions!

### DISCUSSION THREAD

**Discussion by timtnleeProject** [timtnleeProject/vuejs-clipper#85] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by davidbonting** [comment]
Hi @timtnleeProject,

I have been using your product since the beginning (3 years) and I enjoy using it. My users experience the tool as very pleasant. Thanks for all the work over the years.

Despite the fact that I adjust the product to my wishes, after 3 years there is something I can't figure out. Not even after 2-3 days of playing with RXJS (which I'm really not familiar with at all).

I would like to see a "transform tools" option in the clipper-fixed version. Just like you have with clipper-basic:
![image](https://user-images.githubusercontent.com/6029739/130044163-d2b59f25-134f-4366-8488-5c6339a89c41.png)

![ice_video_20210819-111236_edit_0_1_](https://user-images.githubusercontent.com/6029739/130044201-4ab7643c-f53d-470b-8d36-a15b77e94fa6.gif)


This is how far I've come by looking at your code etc (with the recent update of scaling within clipper-fixed helping a lot). The problem that remains is that the image moves when I move. So at the moment of such a "transform-tool corner" click I would not want to move anymore, but start scaling.

The problem is that I just can't seem to get the drag events to stop at that point.

I'm not going to ask you to build this, you do this voluntarily. I'm just dumping it here so others may be able to help. 

I can continue with these 2 solutions: 
1. Clipper-basic "transform-tools" as an option in clipper-fixed
2. An option to temporarily stop drag events

**Reply by timtnleeProject** [reply]
Hi @davidbonting ,
instead of stopping the drag event, we should make **moving** and **transform** action two different events.
Basically, you'll detect the target element of the mousedown/touchdown event, if it's on the border or the corner, it's the **transform** event.
Otherwise, if the element is the image then it's the **moving** event.
